core/src/test/java/org/apache/calcite/test/SqlFunctionsTest.java

```diff
@@ -76,6 +78,34 @@
     assertEquals("a bcd iijk", lower("A bCd Iijk"));
   }
 
+  @Test public void testFromBase64() {
```

Comment:
Can you also add to/from tests ?  Presumably `fromBase64(toBase64(x)) == x`. 
```java
// add more examples as you see fitted
for(String expected: Arrays.asList("", "0", "a", " ", "\n", "\r\n", "\u03C0")) {
  String result = fromBase64(toBase64(code));
  assertEquals(result, expected);
}
```

Reply:
I have added the to/from tests.